### PR TITLE
Documentserver helm package

### PR DESCRIPTION
**Pull request by tobiaschielens**

### CODE REVIEW

**Review by m-motawea — CHANGES_REQUESTED** [review]
please add `readme` to explain the required config for the chart

you can use https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/kube-prometheus-stack/README.md as an example

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
meeting chart in another PR, Can we remove it from this one?

can you please follow these [steps](https://github.com/threefoldtech/vdc-solutions-charts#addingmodifying-a-chart-in-the-repo) for adding the chart?

**Comment by tobiaschielens** [comment]
Added readme + removed meetings + added resource limits

**Comment by tobiaschielens** [comment]
Added the steps from https://github.com/threefoldtech/vdc-solutions-charts#addingmodifying-a-chart-in-the-repo

File: charts/documentserver/values.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: NodePort
+  port: 80
+
+ingress:
```
Comment (ashraffouda) [inline charts/documentserver/values.yaml:47]: do we need to expose this solution as well?
if so plz refere to the comment on filebrowser pr
https://github.com/threefoldtech/vdc-solutions-charts/pull/4#discussion_r551987193

**Comment by ken-de-moor** [comment]
To be reviewed

**Comment by tobiaschielens** [comment]
The package works on the threefold vdc
helm install documentserver ./ --set ingress.hosts[0]=helmtest.jimber.io --set ingress.certresolver=gridca

**Pull request closed without merging**